service-front/module/Application/src/Services/SiriusApiService.php

```diff
@@ -162,17 +163,18 @@ public function abandonCase(array $data, Request $request): array
 
      /**
      * @param array $caseDetails
-     * @param string $systemType
+     * @param SiriusDocument $systemType
      * @param Request $request
-     * @param string $base64suffix
+     * @param string $pdfSuffixBase64 Optional. PDF file in base-64 format, if provided
+     *                                will be added to the generated letter.
      * @return array
      * @throws GuzzleException
      */
-    public function sendPdf(
+    public function sendDocumentRequest(
```

Comment:
Nit, but I don't think `Request` adds much here?
```suggestion
    public function sendDocument(
```